cmd/stack.go

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
```

Comment:
As discussed, we may want to create a separate package for stack commands.

Reply:
Yes, I'm thinking we should have a sub directory (package) something like cmd/stacktools

@Bazif-Khan and @Kamran64 maybe we can do a screenshare Monday to reorg this.